Sources/MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -189,8 +185,8 @@ open class RouteProgress: Codable {
      - parameter routeIndex: The index of the route in the `RouteResponse`. By default, the route is the first route in the response.
      - parameter legIndex: Zero-based index indicating the current leg the user is on.
      */
-    public init(route: Route, routeIndex: Int, options: RouteOptions, legIndex: Int = 0, spokenInstructionIndex: Int = 0) {
```

Comment:
We need to update documentation here as well. `routeIndex` is no longer present. `options` and `spokenInstructionIndex` will have to be added as well.